genesyscloud/util/stringmap/stringmap_test.go

```diff
@@ -0,0 +1,33 @@
+package stringmap
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestUnitMergeMaps(t *testing.T) {
+	m1 := map[string][]int{
+		"key1": {1, 2, 3},
+		"key2": {4, 5, 6},
+	}
+
+	m2 := map[string][]int{
+		"key3": {7, 8, 9},
+		"key4": {10, 11, 12},
+	}
+
+	result := MergeMaps(m1, m2)
+
+	// Check if the result contains all the keys from m1 and m2
```

Comment:
This is an ok basic test, but I'd rather see a few different examples of merges (empty, 1, many, etc) on both sides.

Reply:
Added new test case for this